src/main/java/uk/gov/hmcts/reform/sscs/factory/CcdNotificationWrapper.java

```diff
@@ -67,6 +79,16 @@ public SscsCaseData getOldSscsCaseData() {
         return responseWrapper.getOldSscsCaseData();
     }
 
+    @Override
+    public List<SubscriptionWithType> getSubscriptionsBasedOnNotificationType() {
+        List<SubscriptionWithType> subscriptionWithTypeList = new ArrayList<>();
+        subscriptionWithTypeList.add(new SubscriptionWithType(getAppellantSubscription(), APPELLANT));
+        if (SYA_APPEAL_CREATED_NOTIFICATION.getId().equals(getNotificationType().getId())) {
+            subscriptionWithTypeList.add(new SubscriptionWithType(getRepresentativeSubscription(), REPRESENTATIVE));
```

Comment:
Representative subscription may not exist for all appeals. So a check should be made for representative.

Reply:
good catch, @sathishbabuj . I think it's not breaking the app as it creates even if it is null. But I am confirming it now and making sure it's covered by a junit test.

Reply:
the subscription object is never null -> https://github.com/hmcts/sscs-track-your-appeal-notifications/blob/82ae894afb6c0da4cd558e340a6e6d10b22265cb/src/main/java/uk/gov/hmcts/reform/sscs/deserialize/SscsCaseDataWrapperDeserializer.java#L316

Anyway, added junit coverage here.

Reply:
fixed